### PR TITLE
Update uk lang

### PR DESCRIPTION
**Pull request by burunduk**
Please, update ukrainian language.

### CODE REVIEW

**Comment by srowen** [comment]
Sure, this is good. There is one more change to be made though. In `LocaleManager.TRANSLATED_HELP_ASSET_LANGUAGES`, add "uk" to the set of translated languages.

File: android/res/values-uk/strings.xml [hunk 1]
```diff
@@ -85,42 +85,42 @@
...
   <string name="preferences_device_bug_workarounds_title">Виправлення помилок пристрою</string>
-  <string name="preferences_disable_barcode_scene_mode_title">Немає режимі штрих-коду сцени</string>
+  <string name="preferences_disable_barcode_scene_mode_title">Немає режиму штрих-коду сцени</string>
   <string name="preferences_disable_continuous_focus_summary">Стандартний режим фокусування</string>
-  <string name="preferences_disable_continuous_focus_title">Відклюти безперервне фокусування</string>
-  <string name="preferences_disable_exposure_title">Відключити експозицію</string>
+  <string name="preferences_disable_continuous_focus_title">Вимкнути постійне фокусування</string>
+  <string name="preferences_disable_exposure_title">Вимкнути експозицію</string>
   <string name="preferences_disable_metering_title">Немає обліку</string>
+  <string name="preferences_front_light_summary">Покращує розпізнавання при поганому освітлення, але може викликати відблиски. Працює не на всіх пристроях.</string>
```
Comment (srowen) [inline android/res/values-uk/strings.xml]: Would you mind moving these two strings back down by 3 lines? that keeps them in sorted order.

**Comment by burunduk** [comment]
How to add «google.com.ua for UA» in search_country?

**Comment by srowen** [comment]
Ah yes, also update `GOOGLE_COUNTRY_TLD` in the same file.

**Comment by srowen** [comment]
Thanks! will add you to AUTHORS

**Pull request merged**